src/Fl_Tree_Prefs.cxx

```diff
@@ -22,74 +22,26 @@
 
 #include "Fl_System_Driver.H"
 #include <FL/Fl.H>
-#include <FL/Fl_Pixmap.H>
+#include <FL/Fl_Image.H>
 #include <FL/Fl_Tree_Prefs.H>
-
-// INTERNAL: BUILT IN OPEN/STOW XPMS
-//    These can be replaced via prefs.openicon()/closeicon()
-//
+#include <FL/fl_draw.H>
 
 /**
  \cond DriverDev
  \addtogroup DriverDeveloper
  \{
  */
 
-const char * const Fl_System_Driver::tree_open_xpm[] = {
-  "11 11 3 1",
-  ".    c #fefefe",
-  "#    c #444444",
-  "@    c #000000",
-  "###########",
-  "#.........#",
-  "#.........#",
-  "#....@....#",
-  "#....@....#",
-  "#..@@@@@..#",
-  "#....@....#",
-  "#....@....#",
-  "#.........#",
-  "#.........#",
-  "###########"
-};
-
-const char * const Fl_System_Driver::tree_close_xpm[] = {
-"11 11 3 1",
-".      c #fefefe",
-"#      c #444444",
-"@      c #000000",
-"###########",
-"#.........#",
-"#.........#",
-"#.........#",
-"#.........#",
-"#..@@@@@..#",
-"#.........#",
-"#.........#",
-"#.........#",
-"#.........#",
-"###########"
-};
-
-
-/**
- Return the address of a pixmap that show a plus in a box.
-
- This pixmap is used to indicate a brach of a tree that is closed and
- can be opened by clicking it.
-
- Other platforms may use other symbols which can be reimplemented in the
- driver. Notably, Apple Mac systems mark a closed branch with a triangle
- pointing to the right, and an open branch with a triangle pointing down.
- */
-Fl_Pixmap *Fl_System_Driver::tree_openpixmap() {
-  static Fl_Pixmap *pixmap = new Fl_Pixmap(tree_open_xpm);
-  return pixmap;
-}
-
-Fl_Pixmap *Fl_System_Driver::tree_closepixmap() {
-  static Fl_Pixmap *pixmap = new Fl_Pixmap(tree_close_xpm);
-  return pixmap;
+// Draw non-OS specific Fl_Tree open/close icons
+//    ┌───┐   ┌───┐
+//    │ + │   │ - │
+//    └───┘   └───┘
+void Fl_System_Driver::tree_draw_expando_button(int x, int y, bool state, bool active) {
+  fl_rectf(x, y, 11, 11, FL_BACKGROUND2_COLOR);         // fill
```

Comment:
Removing the ternary for the fill color is not a good change, it messes up the color when the tree is disabled and the system uses a colorful theme (before/after):
![bez tytułu](https://github.com/fltk/fltk/assets/23746867/6ed11815-8062-409a-bf51-ce941d1dbbae)
I'll revert that change and push.